app/v1/dist/index.html

```diff
@@ -222,9 +226,23 @@
           viewerElement.appendChild(renderer.domElement);
           gui = new dat.GUI({ autoPlace: false });
           viewerElement.appendChild(gui.domElement);
+          
+          viewerElement.appendChild(makeDownloadButton());
+          
           addPage(viewerElement);
           window.addEventListener( 'resize', onWindowResize, false );
         }
+        function makeDownloadButton() {
+          downloadButton = document.createElement("button");
+          downloadButton.setAttribute("type", "button");
+          downloadButton.appendChild(document.createTextNode("Download"));
+          downloadButton.setAttribute("style", "position: absolute; top: 2px; right: 2px;");
+          downloadButton.addEventListener("click", () => {
+            const blob = new Blob([readFileSync(path).data()], {type: "text/plain;charset=utf-8"});
```

Comment:
I'm pretty sure you need to remove the .data() from here, too. :)

Reply:
Indeed I do 😬